.eslintrc

```diff
@@ -64,5 +64,12 @@
         ]
       }
     ]
-  }
+  },
+  "overrides": [
+    {
+      // This rule prevents commiting because of we have used page.getBy...
+      "files": ["e2e/**"],
+      "rules" : {"testing-library/prefer-screen-queries": "off"}
+    }
```

Comment:
We need to completely prevent the testing-library from checking the e2e directory, including all its rules.

Something like this (not sure wildcards works for rules): 
```js
{
      files: ['**/e2e/**'],
      rules: {
        'testing-library/*': 'off',
      },
    },
```

Reply:
Doesn't seem like wildcard works for rules, should we individually disable all testing rules in that case?

Reply:
Yeah, unfortunately, I've not found a [permutation](https://github.com/testing-library/eslint-plugin-testing-library?tab=readme-ov-file#eslint-overrides) that works reliably.